workers/ohsome_quality_analyst/api/api.py

```diff
@@ -122,74 +117,40 @@ def empty_api_response() -> dict:
 
 
 @app.get("/indicator")
-async def get_indicator(
-    name: IndicatorEnum,
-    layerName: LayerEnum,
-    includeSvg: bool = False,
-    bpolys: Optional[str] = None,
-    dataset: Optional[DatasetEnum] = None,
-    featureId: Optional[str] = None,
-    fidField: Optional[FidFieldEnum] = None,
-):
-    """Create an Indicator.
+async def get_indicator(parameters=Depends(IndicatorGETRequestModel)):
+    """Get an already calculated Indicator for region defined by OQT.
 
-    Either the parameters `dataset` and `featureId` have to be provided
-    or the parameter `bpolys` in form of a GeoJSON.
-
-    Depending on the input, the output is a GeoJSON Feature or
-    FeatureCollection with the indicator results.
-    The Feature properties of the input GeoJSON will be preserved
-    if they do not collide with the properties set by OQT.
+    The response is a GeoJSON Feature with the indicator results as properties.
+    To request an Indicator for a custom AOI pease use the POST method.
     """
-    if bpolys is not None:
-        bpolys = json.loads(bpolys)
-    raw = {
-        "name": name,
-        "layerName": layerName,
-        "includeSvg": includeSvg,
-        "bpolys": bpolys,
-        "dataset": dataset,
-        "featureId": featureId,
-        "fidField": fidField,
-    }
-    parameters = {k: v for k, v in raw.items() if v is not None}
     return await _fetch_indicator(parameters)
 
 
 @app.post("/indicator")
-async def post_indicator(
-    parameters: IndicatorRequestModel,
-):
+async def post_indicator(parameters: IndicatorPOSTRequestModel):
     """Create an Indicator.
 
-    Either the parameters `dataset` and `featureId` have to be provided
-    or the parameter `bpolys` in form of a GeoJSON.
```

Comment:
Why is this not valid anymore?

Reply:
It is valid. I forgot to add it again after our discussion to keep the POST request as before.
I will add this to the docstring again